packages/gatsby/cache-dir/head/index.js

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import { Helmet } from "react-helmet-async"
+import { useLocation } from "@reach/router"
+
+export const Head = props => {
+  const location = useLocation()
+
+  const { title, description, image } = props
```

Comment:
Should this be imageSrc or imagePath? We don't want people to pass the full image object.

Reply:
Let's just support both strings and `FileNode`s from gatsby-image! 😎

Reply:
Worth noting that the fallback.src only works for new gatsby-plugin-image and not old. Not sure if we can warn there.